<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="terminate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminate process</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="cpu_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ram_table">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>110</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="storage_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>611</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="network_table">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>610</y>
     <width>611</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>90</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RAM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>590</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="cpu_groupbox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>CPU</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ram_groupbox">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>80</y>
     <width>331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>RAM</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="storage_groupbox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>580</y>
     <width>331</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="network_groupbox">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>580</y>
     <width>331</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Network</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pid_spinbox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>35</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PID</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
